### ISSUE TITLE
{@link ... } at the start of the comment line is showing as error

### ISSUE THREAD

**Issue by vpetrovych** [cesarParra/apexdocs#69]
When I have the following ApexDoc, I get the error below if the `{@link ... }` is at the start of the comment:

```
/**
 * A formatter which formats phone numbers as they are entered.
 *
 * <p>An AsYouTypeFormatter can be created by invoking
 * {@link PhoneNumberUtil#getAsYouTypeFormatter}. After that, digits can be added by invoking
 * {@link #inputDigit} on the formatter instance, and the partially formatted phone number will be
 * returned each time a digit is added. {@link #clear} can be invoked before formatting a new
 * number.
 *
 */
```

```
 2/18/2023, 2:57:45 AM: AsYouTypeFormatter - Doc comment parsing error. Level: Type
 2/18/2023, 2:57:45 AM: Comment:
 /**
 * A formatter which formats phone numbers as they are entered.
 *
 * <p>An AsYouTypeFormatter can be created by invoking
 * {@link PhoneNumberUtil#getAsYouTypeFormatter}. After that, digits can be added by invoking
 * {@link #inputDigit} on the formatter instance, and the partially formatted phone number will be
 * returned each time a digit is added. {@link #clear} can be invoked before formatting a new
 * number.
 *
 */
 2/18/2023, 2:57:45 AM: Exception: Error parsing Apex doc. Line 5:3 - mismatched input '{@' expecting {NEWLINE, SPACE, JAVADOC_END}
 2/18/2023, 2:57:45 AM: =================================
```

But if I add anything else before the `{@link ... }`, then the error disappears:

```
/**
 * A formatter which formats phone numbers as they are entered.
 *
 * <p>An AsYouTypeFormatter can be created by invoking
 * test {@link PhoneNumberUtil#getAsYouTypeFormatter}. After that, digits can be added by invoking
 * test {@link #inputDigit} on the formatter instance, and the partially formatted phone number will be
 * returned each time a digit is added. {@link #clear} can be invoked before formatting a new
 * number.
 *
 */
```

**Comment by cesarParra** [comment]
Thanks @vpetrovych . I'll investigate

**Comment by cesarParra** [comment]
I just released version 2.13.1 which should hopefully fix the issue. Let me know otherwise.
